pkg/utils/overriding/test-fixtures/patches/add-command-and-component/patch.yaml

```diff
@@ -4,6 +4,7 @@ commands:
     id: "commandWithTypeChanged"
   - exec:
       commandLine: "commandToReplaceCommandLineReplaced"
+      component: "mycomponent"
```

Comment:
You should not have to add the component in the `patch.yaml`, precisely because it's expected to be an override, and overrides have a dedicated schema that makes most fields optional.

But of course the `original.yaml` and `result.yaml` will need it.

Reply:
Removed the component